Src/library/net/Clib/network.c

```diff
@@ -726,26 +726,36 @@ EIF_INTEGER c_peer_name(EIF_INTEGER s, EIF_POINTER addr, EIF_INTEGER length)
 
 /*x basic types receiving (macro) */
 #define CREADX(bytes_read, descriptor, element_pointer, sizeofelement, flags) \
-	eif_net_check (bytes_read = recv((EIF_SOCKET_TYPE) descriptor, (char *) element_pointer, (int) sizeofelement, (int) flags));
+	bytes_read = recv((EIF_SOCKET_TYPE) descriptor, (char *) element_pointer, (int) sizeofelement, (int) flags)
 
 /*x basic types sending (macro) */
+#define CSENDX(bytes_sent, descriptor, element_pointer, sizeofelement, flags) \
+	bytes_sent = send((EIF_SOCKET_TYPE) descriptor, (char *) element_pointer, (int) sizeofelement, (int) flags)
+
 #define CPUTX(descriptor,element_pointer,sizeofelement) \
-	eif_net_check (send ((EIF_SOCKET_TYPE) descriptor, (char *) element_pointer, (int) sizeofelement, (int) 0));
+	eif_net_check (send((EIF_SOCKET_TYPE) descriptor, (char *) element_pointer, (int) sizeofelement, (int) 0));
 
 /*x basic types sending TO (macro) */
 #define CSENDXTO(descriptor,element_pointer,sizeofelement, flags, addr_pointer, sizeofaddr) \
 	eif_net_check (sendto ((EIF_SOCKET_TYPE) descriptor, (char *) element_pointer, \
 		(int) sizeofelement, (int) flags, (struct sockaddr *) addr_pointer, (int) sizeofaddr));
 
+
+/******************************************************/
+/* send to operations                                 */
+/******************************************************/
+
 void c_send_char_to(EIF_INTEGER fd, EIF_CHARACTER c, EIF_INTEGER flags, EIF_POINTER addr_pointer, EIF_INTEGER sizeofaddr)
 	/*x transmission of character c through socket fd */
 {
+	/* Unused (!) by EiffelNet */
```

Comment:
We can remove it because this is actually only used by the obsolete version of EiffelNet which has its own C code.